### ISSUE TITLE
Has anyone reproduced the reported performance in the paper?

### ISSUE THREAD

**Issue by dingli-dean** [fmthoker/skeleton-contrast#7]
I have tried to reproduce the performance, but limited success was achieved. Only got  67.20% acc with intra-skeleton contrast of graph-based network, while the corresponding performance is 72.5% in table 6 of the paper.

**Comment by fmthoker** [comment]
can you upload  the training config and logs?

**Comment by dingli-dean** [comment]
Self-supervised training config:
{'graph_args': {'labeling_mode': 'spatial'},
 'num_class': 128,
 'num_person': 2,
 'num_point': 25}

{'data_path': '/media/liding/dataset/NTU/data_Skeleton_Contrast/NTU-RGB-D-60-AGCN/xview/train_data_joint.npy',
 'input_representation': 'graph-based',
 'input_size': 64,
 'l_ratio': [0.1, 1],
 'num_frame_path': '/media/liding/dataset/NTU/data_Skeleton_Contrast/NTU-RGB-D-60-AGCN/xview/train_num_frame.npy'}{'batch_size': 256,
 'checkpoint_path': '/media/liding/experiments/sk/exp_g-2/ntu_60_cross_view/intraskeleton_graph_based',
 'cos': False,
 'dev': 33,
 'epochs': 451,
 'gpu': 0,
 'lr': 0.01,
 'mlp': True,
 'moco_dim': 128,
 'moco_k': 16384,
 'moco_m': 0.999,
 'moco_t': 0.07,
 'momentum': 0.9,
 'pre_dataset': 'ntu60',
 'print_freq': 10,
 'protocol': 'cross_view',
 'resume': '',
 'schedule': [351],
 'seed': None,
 'skeleton_representation': 'graph-based',
 'start_epoch': 0,
 'weight_decay': 0.0001,
 'workers': 32}

Self-supervised training tensorboard log is:
![g_1_ssl_traning](https://user-images.githubusercontent.com/11832829/161181792-55ba9d78-97e6-48d1-bd8f-3962a74e34b3.png)


Linear evaluation config is:
-------agcn_model_args-------
{'graph_args': {'labeling_mode': 'spatial'},
 'num_class': 60,
 'num_person': 2,
 'num_point': 25}
-------train_feeder_args-------
{'data_path': '/home/liding/dataset/NTU/data_Skeleton_Contrast/NTU-RGB-D-60-AGCN/xview/train_data_joint.npy',
 'input_representation': 'graph-based',
 'input_size': 64,
 'l_ratio': [0.5, 1.0],
 'label_path': '/home/liding/dataset/NTU/data_Skeleton_Contrast/NTU-RGB-D-60-AGCN/xview/train_label.pkl',
 'num_frame_path': '/home/liding/dataset/NTU/data_Skeleton_Contrast/NTU-RGB-D-60-AGCN/xview/train_num_frame.npy'}
-------exp_args-------
{'batch_size': 1024,
 'checkpoint_path': '/media/liding/experiments/sk/exp_g-2/ntu_60_cross_view/intraskeleton_graph_based/action_cls_joint',
 'dev': 31,
 'epochs': 80,
 'finetune_dataset': 'ntu60',
 'finetune_skeleton_representation': 'graph-based',
 'gpu': 0,
 'lr': 0.1,
 'momentum': 0.9,
 'pretrain_skeleton_representation': 'graph-based',
 'pretrained': '/media/liding/experiments/sk/exp_g-2/ntu_60_cross_view/intraskeleton_graph_based/checkpoint_0450.pth.tar',
 'print_freq': 10,
 'protocol': 'cross_view',
 'resume': '',
 'schedule': [50, 70],
 'seed': None,
 'start_epoch': 0,
 'weight_decay': 0.0,
 'workers': 32}

**Comment by dingli-dean** [comment]
It is the batch_size in linear classifier training that hinders the reproduction of experiment. 
Do not modify defaut number in .sh file.
Set the batch_size == 64.
